CHANGELOG.md

```diff
@@ -8,6 +8,27 @@ This project follows [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) fo
 
 ## [Unreleased]
 
+## [1.15.0] - 2023-03-20
+
+### Added
+* Feature: Add DynamoDB provider to parameters module (#1091) by @scottgerring
+* Feature: Improve powertools-cloudformation API (#1082) by @msailes
```

Comment:
Feature: Update to powertools-cloudformation to deprecate `Response.success()` and `Response.failed()` methods. New helper methods are added to make it easier to follow best practices `Response.success(String physicalResourceId)` and `Response.failed(String physicalResourceId)`. For a detailed explanation please read the [powertools-cloudformation documentation page](https://awslabs.github.io/aws-lambda-powertools-java/utilities/custom_resources/).